torch/_inductor/utils.py

```diff
@@ -855,15 +855,18 @@ def fresh_inductor_cache(cache_entries=None, dir=None, delete=True):
                             }
                         )
         if delete:
-            shutil.rmtree(inductor_cache_dir)
+            shutil.rmtree(
+                inductor_cache_dir,
+                # Let's not fail if we can't clean up the temp dir. Also note that for
+                # Windows, we can't delete the loaded modules because the module binaries
+                # are open.
+                onerror=lambda func, path, exc_info: log.warning(
+                    "Failed to remove temporary cache dir at %s", inductor_cache_dir
+                ),
+            )
     except Exception:
-        if not _IS_WINDOWS:
-            """
-            Windows can't delete the loaded modules, because the modules binaries are opened.
-            TODO: discuss if have better solution to handle this issue.
-            """
-            log.warning("on error, temporary cache dir kept at %s", inductor_cache_dir)
-            raise
+        log.warning("on error, temporary cache dir kept at %s", inductor_cache_dir)
```

Comment:
```suggestion
        log.warning("on error, temporary cache dir kept at %s", inductor_cache_dir, exc_info=True)
```
? want to include error info?

Reply:
@Skylion007 I wouldn't think we'd want to here. The exception is unrelated to the rmtree and is being raised. I think the intention here is just to let you know where the inductor artifacts are located for debugging purposes?